### PR TITLE
Finestra Miner branding requirements added.

### PR DESCRIPTION
**Pull request by encku**

### CODE REVIEW

**Comment by tyler-whitman** [comment]
Once you have a maker account you will also need to add your support email here https://github.com/helium/hotspot-app/blob/main/src/utils/stakingClient.ts#L55

**Review by tyler-whitman — APPROVED** [review]
Code looks good and is working as expected, just missing the support email once you have a maker ID. 

I will wait for one other approver as well.

**Comment by cokes518** [comment]
FYI, we'll merge this to main when community has approved Smart Mimic as a Maker.

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Once you have a maker account you will also need to add your support email here https://github.com/helium/hotspot-app/blob/main/src/utils/stakingClient.ts#L55

**Comment by cokes518** [comment]
As part of assert_v2, you will also want to add your Hotspot and its default antenna dBi:
https://github.com/helium/hotspot-app/blob/da5fbce2608fb3a23347ffbde556d2829dce9f29/src/components/HotspotConfigurationPicker.tsx#L26

and the EN string
https://github.com/helium/hotspot-app/blob/da5fbce2608fb3a23347ffbde556d2829dce9f29/src/locales/en.ts#L959

**Review by tyler-whitman — CHANGES_REQUESTED** [review]
Thanks for adding the support email. The last change would be the antenna info with Assert V2 that Coco mentioned

**Comment by tyler-whitman** [comment]
You can reference https://github.com/helium/hotspot-app/pull/497 for the antenna changes

**Comment by encku** [comment]
> You can reference #497 for the antenna changes



> Thanks for adding the support email. The last change would be the antenna info with Assert V2 that Coco mentioned. Since this is a new change. I can also do it for you if you give me the values you would like me to add.

#497 completed.

**Comment by cokes518** [comment]
@tyler-whitman this can be merged

File: src/components/HotspotConfigurationPicker.tsx
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:18]: You need to remove these lines, 18 - 48.

They got moved to the `constants/antennas` file

File: src/components/HotspotPairingList.tsx
Comment (tyler-whitman) [inline src/components/HotspotPairingList.tsx:28]: Finestra should be below LongAP please. We keep them in the order they were added.

File: src/components/HotspotConfigurationPicker.tsx
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:30]: You need to move this to https://github.com/helium/hotspot-app/blob/main/src/constants/antennas.ts#L13

File: src/components/HotspotConfigurationPicker.tsx
Comment (tyler-whitman) [inline src/components/HotspotConfigurationPicker.tsx:45]: You need to move this to https://github.com/helium/hotspot-app/blob/main/src/constants/antennas.ts#L30

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

File: src/features/hotspots/setup/HotspotSelectionListItem.tsx
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSelectionListItem.tsx:60]: I must have missed this one, sorry one last change this needs to be below `LONGAPONE`

File: src/features/hotspots/setup/HotspotSetupDiagnosticsScreen.tsx
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSetupDiagnosticsScreen.tsx:39]: Same with this one

**Review by tyler-whitman — CHANGES_REQUESTED** [review]

**Comment by tyler-whitman** [comment]
I made some fixes so I opened it as another PR. https://github.com/helium/hotspot-app/pull/619

**Comment by tyler-whitman** [comment]
I made some fixes so I opened it as another PR. https://github.com/helium/hotspot-app/pull/619

**Pull request closed without merging**
